postgresql/src/storages/postgres/detail/connection_impl.cpp

```diff
@@ -457,8 +494,7 @@ Connection::StatementId ConnectionImpl::PortalBind(
   TimeoutDuration network_timeout = ExecuteTimeout(statement_cmd_ctl);
   auto deadline = testsuite_pg_ctl_.MakeExecuteDeadline(network_timeout);
   SetStatementTimeout(std::move(statement_cmd_ctl));
-
-  tracing::Span span{scopes::kQuery};
+  tracing::Span span{scopes::kBind + ":" + portal_name};
```

Comment:
please make sure that all pg spans start from the same prefix, otherwise it would be hard to find all of them in traces

Reply:
```
kPrepare = "pg_prepare"
kExec = "pg_exec"
kBind = "pg_bind"
kQuery = "pg_query"
```
they all have the same "pg_" prefix so I don't think it's a problem. I'd like to distinguish bind from exec of portal, prepare and actual querying.